Bottom: 9cb18b0152e07aa6e793551ca74d71f4c2cf6fd3
Top:    6ade27af770937b5cf95e0463911d2238046554e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 20:13:40 +0100

Refresh of 3d-mouse-cursor-and-other-proj

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 3715e05..c79673f 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -236,7 +236,7 @@ ghid_get_coords (const char *msg, int *x, int *y)
     }
 }
 
-float
+static float
 determinant_2x2 (float m[2][2])
 {
   float det;
@@ -245,7 +245,8 @@ determinant_2x2 (float m[2][2])
   return det;
 }
 
-float
+#if 0
+static float
 determinant_4x4 (float m[4][4])
 {
   float det;
@@ -263,8 +264,9 @@ determinant_4x4 (float m[4][4])
         m[0][1] * m[1][0] * m[2][2] * m[3][3]+m[0][0] * m[1][1] * m[2][2] * m[3][3];
    return det;
 }
+#endif
 
-void
+static void
 invert_2x2 (float m[2][2], float out[2][2])
 {
   float scale = 1 / determinant_2x2 (m);
@@ -274,7 +276,8 @@ invert_2x2 (float m[2][2], float out[2][2])
   out[1][1] =  m[0][0] * scale;
 }
 
-void
+#if 0
+static void
 invert_4x4 (float m[4][4], float out[4][4])
 {
   float scale = 1 / determinant_4x4 (m);
@@ -328,6 +331,7 @@ invert_4x4 (float m[4][4], float out[4][4])
                m[0][2] * m[1][0] * m[2][1] - m[0][0] * m[1][2] * m[2][1] -
                m[0][1] * m[1][0] * m[2][2] + m[0][0] * m[1][1] * m[2][2]) * scale;
 }
+#endif
 
 
 void
